server/chat-plugins/tcgtabletop.ts

```diff
@@ -6,33 +6,15 @@
 */
 
 
-import * as https from 'https';
+import {Net} from '../../lib/net';
 import * as querystring from 'querystring';
 
 
 const SEARCH_PATH = '/api/v1/Search/List/';
 const DETAILS_PATH = '/api/v1/Articles/Details/';
 
 async function getFandom(site: string, pathName: string, search: AnyObject) {
-	const reqOpts = {
-		hostname: `${site}.fandom.com`,
-		method: 'GET',
-		path: `${pathName}?${querystring.stringify(search)}`,
-		headers: {
-			'Content-Type': 'application/json',
-		},
-	};
-
-	const body: any = await new Promise((resolve, reject) => {
-		https.request(reqOpts, res => {
-			if (!res.statusCode || !(res.statusCode >= 200 && res.statusCode < 300)) return reject(new Error(`Not found.`));
-			const data: string[] = [];
-			res.setEncoding('utf8');
-			res.on('data', chunk => data.push(chunk));
-			res.on('end', () => resolve(data.join('')));
-		}).on('error', reject).setTimeout(5000).end();
-	});
-
+	const body = await Net(`https://${site}.fandom.com/${pathName}?${querystring.stringify(search)}`).get();
```

Comment:
this is missing a `Content-Type` `application/json` header, which might be relevant.

Reply:
Looks like it's not. @Asheviere, do you know why you originally sent it? `Content-Type` is a response header, not a request header; the corresponding request header is `Accept` to list which content types you support.